pegasus/demo/demo_workflow.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import sys
+from pathlib import Path
+from Pegasus.api import *
+
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+home = Path.home().absolute()
+shared_scratch_dir = f"{home}/workflows/scratch"
+local_storage_dir = f"{home}/workflows/output"
+sc = SiteCatalog()
+sc.add_sites(
+    Site("local", arch=Arch.X86_64, os_type=OS.LINUX).add_directories(
+        Directory(Directory.SHARED_SCRATCH, shared_scratch_dir).add_file_servers(
+            FileServer("file://" + shared_scratch_dir, Operation.ALL)
+        ),
+        Directory(Directory.LOCAL_STORAGE, local_storage_dir).add_file_servers(
+            FileServer("file://" + local_storage_dir, Operation.ALL)
+        ),
+    )
+)
+
+saga = Site("saga", arch=Arch.X86_64, os_type=OS.LINUX)
+
+saga.add_directories(
+    Directory(Directory.SHARED_SCRATCH, shared_scratch_dir).add_file_servers(
+        FileServer("file://" + shared_scratch_dir, Operation.ALL)
+    )
+)
+
+saga.add_env(
+    key="PEGASUS_HOME", value="/nas/gaia/shared/cluster/pegasus5/pegasus-5.0.0"
+)
+
+saga.add_pegasus_profile(style="glite", auxillary_local=True, data_configuration="sharedfs")
+saga.add_condor_profile(grid_resource="batch slurm")
+
+sc.add_sites(saga)
+
+# Container
+py36_container = Container(
+            "python36",
+            Container.DOCKER,
+            image="/nas/gaia/users/napiersk/archives/docker/python-3-6-3.tar",
+            image_site="saga"
+# ?other args: arguments, mounts, checksum, metadata, bypass_staging
+        )
+
+#saga31_request = {"run_on_single_node": "saga31", "partition": "gaia"}
+
+tc = TransformationCatalog().add_containers(py36_container)
+
+py36_version = Transformation(
+             "py36_version",
+             pfn="/usr/local/bin/python",
+             site="saga",
+             container=py36_container,
+# TODO: need to add resource request:?  resource_request=saga31_request,
+# needed? type="INSTALLED",
+)             
```

Comment:
```suggestion
py36_version = Transformation(
             "py36_version",
             pfn="/usr/local/bin/python",
             site="saga",
             container=py36_container,
             is_stageable=false,
# TODO: need to add resource request:?  resource_request=saga31_request,
# needed? type="INSTALLED",
)             
```

`is_stageable` needs to be set to false to treat the PFN as installed on the docker container and not a PFN which is on SAGA to be transferred into the container. Ref on [Pegasus Documentation](https://pegasus.isi.edu/documentation/user-guide/containers.html#configuring-workflows-to-use-containers)

Reply:
Thanks @LichMaster98 I have that change now at: 
https://github.com/GAIA-AIDA/aida-integration/pull/22/files#diff-f867d610acddad8f8ea58050228717c6f8c62187b6768d215b9d8026ffe75f8dR46

Now hitting new error, in comment above.

Reply:
@danielnapierski Can you run `pegasus-analyzer /path/to/workflow/dir` (the same path you would use `pegasus-status` with) and see if that gives any additional information than the error out log? The analyzer doesn't always provide useful information (it'll probably just confirm what we currently know the error is) but sometimes it's helpful. Otherwise, the configuration at this point looks correct to me so we may need to reach out to the Pegasus team for more advice.